CHANGELOG.md

```diff
@@ -20,28 +20,28 @@ Solidity optimizer: `disabled`
 
 ### Core contracts
 
--   `Safe` at `0x41675C099F32341bf84BFc5382aF534df5C7461a`
--   `SafeL2` at `0x29fcB43b46531BcA003ddC8FCB67FFE91900C762`
+-   `Safe` at `0xCDaf5D62B2bA00C9A5D7e85983055B4168db16B3`
```

Comment:
Also, FYI, when zksync gets merged, we'll have to update the changelog with ZkSync addresses too

Reply:
Can we just keep this `TBD` until 1.5.0 is finalized? It will change again when we do an audit and have to modify the contracts as well.